packages/devextreme/testing/tests/DevExpress.data/queryArray.tests.js

```diff
@@ -350,6 +350,24 @@ QUnit.test('filter with functional getter', function(assert) {
     });
 });
 
+QUnit.test('filter with undefined "langParams"', function(assert) {
```

Comment:
Let's add one more QUnit test with the case ```sensitivity: 'base'```
It's supported by the ```DataQuery``` [here](https://github.com/DevExpress/DevExtreme/pull/27662/files#diff-31a735c2b79c2deb183db2400f94c57fbe4c05d459afc8cafb6121f22bf5dedbR195)
[Link to the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#base)

This test may help us in future code modifications.
With it, we don't forget that we support the ```sensitivity: 'base'``` :)